### ISSUE TITLE
process embedded twitter content properly

### ISSUE THREAD

**Issue by anarcat** [wallabag/wallabag#3427]
### Issue details

Wallabag doesn't deal well with articles that have embedded Twitter links. For example, [this post on Medium.com](https://medium.com/@dmytri/that-time-davidgraeber-called-me-an-asshole-fictitious-politics-socialism-in-reverse-and-ubi-on-a91fca63a1d1) embeds a lot of Twitter content which is crucial to the understanding of the piece.

The rendering of this by wallabag is pretty bad. The first few paragraphs of the ePUB show up like this:

```
That Time @DavidGraeber Called Me An Asshole: Fictitious Politics, Socialism in Reverse, and UBI on Twitter.

[embedded content]
It’s not everyday a best-selling writer and leading anthropologist calls you an asshole. UBI Twitter is a hell of a place, where valid arguments are not required, and things tend to escalate quickly.
This all started innocently enough, @mempko posted a message asking @davidgraeber what he thought of my critique of UBI.

[embedded content]
I responded with a link to my essay, which best expresses my views on the subject.

[embedded content]
[...]

```

The HTML version is not much better: instead of the `[embedded content]` warnings, we get... nothing at all: just the blank space.

It would be great if Wallabag could render that content correctly. I understand this may be a problem with the backing library, and `f43.me` has similar problems rendering the content. But from what I understand of the way that tool works, it *can't* handle embedded content at all.. I would assume that's the job of the upstream library to send the right HTML content in the first place...

### Environment

* wallabag version (or git revision) that exhibits the issue: Version: 2.2.2
* How did you install wallabag? unknown
* Last wallabag version that did not exhibit the issue (if applicable): N/A
* php version: ?
* OS: Debian stretch
* type of hosting (shared or dedicated): shared
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MySQL?

### Steps to reproduce/test case

Please provide necessary steps for reproduction of this issue, or better the
reduced test case (without any external dependencies, if possible).

Add [this link](https://medium.com/@dmytri/that-time-davidgraeber-called-me-an-asshole-fictitious-politics-socialism-in-reverse-and-ubi-on-a91fca63a1d1) to your wallabag instance and look at the result.

**Comment by j0k3r** [comment]
Trying on the upcoming 2.3 I don't have that `[embedded content]` (as far as I remember it was a bug in graby).
But an other problem is that all embedded content on Medium are medium iframe which then embed the tweet.

<img width="930" alt="screen 2017-11-21 a 10 03 37" src="https://user-images.githubusercontent.com/62333/33063629-4db01eb0-cea3-11e7-815e-fa967b513f70.png">

And Medium doesn't allow them to be displayed elsewhere but on their website (using [CSP](https://developer.mozilla.org/fr/docs/HTTP/CSP)):

> Refused to display 'https://medium.com/media/682de8e752bd83beece6c35d52693fe0?postId=a91fca63a1d1' in a frame because it set 'X-Frame-Options' to 'sameorigin'.

Which means embed won't be displayed on wallabag.it for example but I guess they will be displayed in app.

![image](https://user-images.githubusercontent.com/62333/33063832-f85e4454-cea3-11e7-9f90-c3690799694f.png)

**Comment by anarcat** [comment]
right, so that's what i get here too. the `[embedded content]` tag is in the ePUB only.
